cmd/internal/dgather.go

```diff
@@ -38,7 +38,7 @@ inside a bosh manifest file.
 			return fmt.Errorf("base directory cannot be empty")
 		}
 
-		namespace := viper.GetString("cf-operator-namespace")
+		namespace := viper.GetString("namespace")
```

Comment:
Why change this as part of this PR? This will probably mess up CI too.
@manno didn't we recently change this?

Reply:
We did, so it's the same everywhere. Do we support multiple namespaces?

Reply:
@viovanov @manno I thought this namespace was to generate bosh-jobs properties and it should not same as cf-operator namespace. For example, I had a cf-operator run in default namespace and a BoshDeployment in cf namespace. Each [instance address](https://github.com/cloudfoundry-incubator/cf-operator/blob/f39e76b13fab2e048d3b4c7d9fa2ee423b33d2fd/pkg/bosh/manifest/manifest.go#L232-L233) should be generated by cf namespace: nats-deployment-nats-0.cf.svc.cluster.local. So cf-operator can handle multiples BoshDeployment instances cross namespaces. That's what I wanted to do.